### DISCUSSION TITLE
Minimizing search results

### DISCUSSION THREAD

**Discussion by aborruso** [squidfunk/mkdocs-material#5817] [Q&A] [answered]
Hi,
is there a way to have in the results only the page title (what I have highlighted in red in the image below)?

Thank you 

![image](https://github.com/squidfunk/mkdocs-material/assets/30607/62606819-3336-4f9a-ab9a-58ccef34741f)

**Comment by squidfunk** [comment, marked as answer]
The search plugin currently doesn't allow for that. As said, we're working on ground up rewrite which will yield much better search summaries and provide more compact results, but it's still going to take some time. However, you have two options:

1. Hide the search result content with CSS. That way, the content will still be searchable, so keywords in the content of a page will still match that page. Something like:

    ``` css
    .md-search-result__article > :not(h1, h2, div) {
       display: none;
    }
    ```

2. Remove the `text` attribute from all entries in `search_index.json` as a post-processing step in a custom hook. It's likely 4-5 lines of Python, but search will only match titles.

**Reply by aborruso** [reply]
Thank you very much

**Reply by aborruso** [reply]
@squidfunk do you know why if I search in homepage, I have no page titles in search preview?

In example in the home no titles  https://hfcqgis.opendatasicilia.it/?q=spatial+join

![image](https://github.com/squidfunk/mkdocs-material/assets/30607/5c18df18-95f8-4cbd-b1fc-6609f982ec54)

Here I have the titles https://hfcqgis.opendatasicilia.it/calcolatore_campi/field_calc/?q=spatial+join

![image](https://github.com/squidfunk/mkdocs-material/assets/30607/f4763c8f-1688-486d-b211-41b6ef682587)


Thank you

**Reply by squidfunk** [reply]
I'm not sure. If you can craft a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), we can look into it. Otherwise, we're not sure if the behavior is caused by our customizations or some combination of configuration of Material for MkDocs.

**Reply by aborruso** [reply]
> I'm not sure. If you can craft a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), we can look into it. Otherwise, we're not sure if the behavior is caused by our customizations or some combination of configuration of Material for MkDocs.

It was my CSS customizations.  Sorry if I asked here, without checking first.

And thanks for everything
